casanovo/config.py

```
@@ -162,6 +164,55 @@ def __init__(self, config_file: Optional[str] = None):
                 " please specify it explicitly in the config file."
             )
 
+    def _builtin_configs(self):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_builtin_configs`

(ANN202)